admin/views.py

```diff
@@ -495,8 +510,17 @@ def resource_create(resource_type):
     attributes_to_save = {}
     for attribute in editable_attributes:
         attribute_value = request.form.get(attribute["name"])
-        validated_attribute_value = validate_resource_attribute(resource_type, attribute, attribute_value)
-        attributes_to_save[attribute['name']] = validated_attribute_value
+        validated_attribute_value = validate_resource_attribute(
+            resource_type, attribute, attribute_value
+        )
+
+        if attribute["name"] == admin_configs["user"]["secret"]:
+            hashed_password = bcrypt.generate_password_hash(
```

Comment:
Instead of calling this function multiple times, can we create a function that will return hashed_password?

Reply:
yes can create a separate function for hash _password.